src/main/kotlin/com/sourcegraph/cody/context/ui/EnhancedContextPanel.kt

```diff
@@ -256,28 +254,17 @@ class EnterpriseEnhancedContextPanel(project: Project, chatSession: ChatSession)
           fun targetForEvent(e: MouseEvent): Any? =
               tree.getClosestPathForLocation(e.x, e.y)?.lastPathComponent
 
-          // We cache the target of the mouse press, so that if the tree expands before the click
-          // event is generated, we can detect the mouse click event is on a different node and
-          // suppress the popup.
-          private var pressedTarget: Any? = null
-
           override fun mousePressed(e: MouseEvent) {
             super.mousePressed(e)
-            pressedTarget = targetForEvent(e)
-          }
-
-          override fun mouseClicked(e: MouseEvent) {
-            var clickTarget = targetForEvent(e)
-            if (e.clickCount == 1 &&
-                e.button == MouseEvent.BUTTON1 &&
-                pressedTarget === clickTarget &&
-                clickTarget is ContextTreeEditReposNode) {
+            if (targetForEvent(e) is ContextTreeEditReposNode) {
```

Comment:
Does this mean that the same action will be invoked on all mouse clicks, regardless of whether it's the left or right (or middle) button? I'm not sure that's a good idea.

Reply:
It is, but I can limit it to left or right clicks.